styles/components/_slate.scss

```diff
@@ -9,10 +9,14 @@
 	&.-plain { background: $brand-white; }
 
   // For consistency
-  h1, h2, h3, h4, h5, h6, p {
+  h1, h2, h3, h4, h5, h6 {
     margin-bottom: $spacer-large;
   }
 
+  p {
+    margin-bottom: $spacer-small;
```

Comment:
yeah, though it override our default `p` structure given by `_type.scss`
![image](https://cloud.githubusercontent.com/assets/12628112/11355594/26f19e3e-9291-11e5-9ad6-9043fed6baae.png)